util/no_destructor.h

```diff
@@ -33,12 +28,11 @@ class NoDestructor {
   NoDestructor& operator=(const NoDestructor&) = delete;
 
   InstanceType* get() {
-    return reinterpret_cast<InstanceType*>(&instance_storage_);
+    return reinterpret_cast<InstanceType*>(+instance_storage_);
```

Comment:
Why is the `+` needed here?

We seem to be able to get away without it in Chromium -- https://source.chromium.org/chromium/chromium/src/+/main:base/no_destructor.h;l=91;drc=ded7717f5252a0c9d9de56bc22483b195d1801e0